enhancements/ingress/host-network-bind-options.md

```diff
@@ -0,0 +1,266 @@
+---
+title: ingress-host-network-binding-options
+
+  authors:
+- "@elbehery"
+ 
+reviewers:
+- "@alebedev87"
+- "@frobware"
+- "@Miciah"
+
+approvers:
+- "@knobunc"
+- "@tjungblu"
+
+
+  creation-date: 2022-01-10
+  
+  last-updated: 2022-01-10
+
+  status: implementable
+
+  see-also:
+
+  replaces:
+
+  superseded-by:
+
+---
+
+# Ingress Host Network Binding Options
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement updates the Ingress Controller API to allow specifying binding options in the HostNetwork strategy and allows running many instances of the ingress controller on the same node using the HostNetwork strategy, therefore avoiding port binding conflicts.
+## Motivation
+
+While utilising the HostNetwork strategy for ingress controllers, the ports 80, 443, 10443, 10444, 1936 on the host are bound by HAProxy for http, https, no_sni, sni and stats correspondingly. 
+
+Those ports might be occupied by other processes which makes it unfeasible to run multiple ingress controllers on the same node with HostNetwork strategy. 
```

Comment:
maybe something like "**These** ports might be occupied by other processes which will make it **impossible** to run multiple..."